[bug]: 画像など非文字列送信時の挙動改善

### 追加連絡先

_No response_

### 発生した不具合

現状はそのまま処理できずに`failure`として処理されている

以下ログ
```log
15:27:13.259 [JDA MainWS-ReadThread] ERROR net.dv8tion.jda.api.JDA - One of the EventListeners had an uncaught exception
java.util.concurrent.CompletionException: java.lang.NullPointerException: Cannot invoke "com.fasterxml.jackson.databind.JsonNode.get(String)" because the return value of "com.fasterxml.jackson.databind.JsonNode.get(String)" is null
        at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
        at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
        at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1770)
        at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760)
        at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
        at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
        at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
        at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
        at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.NullPointerException: Cannot invoke "com.fasterxml.jackson.databind.JsonNode.get(String)" because the return value of "com.fasterxml.jackson.databind.JsonNode.get(String)" is null
        at MochiMochiTalk.commands.CommandDictionary.getDict(CommandDictionary.java:143)
        at MochiMochiTalk.voice.nvoice.EventListenerForTTS.lambda$onMessageReceived$1(EventListenerForTTS.java:104)
        at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
        ... 6 common frames omitted
```

### 期待した挙動

画像投稿時にはイベントをスキップする

### 推定される発生時刻

現在まで

--- a/app/src/main/java/MochiMochiTalk/voice/nvoice/EventListenerForTTS.java
+++ b/app/src/main/java/MochiMochiTalk/voice/nvoice/EventListenerForTTS.java
@@ -98,6 +98,11 @@ public class EventListenerForTTS extends ListenerAdapter {
         }
 
         if(flag && !rawContent.startsWith(App.getStaticPrefix())) {
+            // early return for not speaking media content
+            if(!message.getAttachments().isEmpty()) {
+                log.info("Message contains media content. : {}", content);
+                return;
+            }
             String immutableContent = content;
             CompletableFuture<Optional<String>> dictFetchFuture = CompletableFuture.supplyAsync(() -> {
                 CommandDictionary instance = CommandDictionary.getInstance();

